### PR TITLE
Fixed timezone offset getting for UTC-n timezones

### PR DESCRIPTION
**Pull request by eugene-platov**
These changes fixes the next issue: https://github.com/date-fns/date-fns/issues/1661

### CODE REVIEW

File: src/_lib/getTimezoneOffsetInMilliseconds/index.js [hunk 1]
```diff
@@ -16,9 +16,12 @@ export default function getTimezoneOffsetInMilliseconds(dirtyDate) {
...
   date.setSeconds(0, 0)
   var millisecondsPartOfTimezoneOffset = date.getTime() % MILLISECONDS_IN_MINUTE
+  var hasNegativeUTCOffset = baseTimezoneOffset > 0
 
   return (
     baseTimezoneOffset * MILLISECONDS_IN_MINUTE +
-    millisecondsPartOfTimezoneOffset
+    (hasNegativeUTCOffset && millisecondsPartOfTimezoneOffset !== 0
+      ? MILLISECONDS_IN_MINUTE + millisecondsPartOfTimezoneOffset
+      : millisecondsPartOfTimezoneOffset)
```
Comment (dkozickis) [inline src/_lib/getTimezoneOffsetInMilliseconds/index.js]: Could you please extract this part of code into separate `var`?

**Comment by eugene-platov** [comment]
@dkozickis Well, some tests were failing if they are run in any UTC-n timezone :)
But I'll try, it will require to mock `getTimezoneOffset` somehow

**Comment by dkozickis** [comment]
Oh... I thought Travis does a full run with all timezones @kossnocorp?

**Comment by kossnocorp** [comment]
@dkozickis it actually does: https://travis-ci.org/github/date-fns/date-fns/jobs/662076035#L1015-L1055

@eugene-platov what is the TZ value that causes the failure? It would be great to add some regression tests.

**Comment by eugene-platov** [comment]
@kossnocorp Thank you
Guess the reason that tests in ci are not failing is that they are running in timezones that doesn't have seconds offsets
If I set `America/Los_Angeles` timezone on my computer, all dates before Nov 19th 1883 will get additional minute and some tests will fail
After this fix the are not failing
Screenshot at Mar 16 02-31-16

**Pull request merged**